bench/ssz/merkleize_bench.exs

```diff
@@ -0,0 +1,43 @@
+alias LambdaEthereumConsensus.SszEx
```

Comment:
I would name this one hash_tree_root which is more important than the merkelization part